.github/workflows/performance_metrics_cron.yml

```diff
@@ -109,10 +100,9 @@ jobs:
         continue-on-error: true
         run: |
           set -euo pipefail
-          cd tests && go test -v -json -count=1 -cover -timeout 6h -parallel ${{ env.TESTPARALLELISM }} . 2>&1 | tee /tmp/gotest.log | gotestfmt
+          cd tests/perf && go test -v -json -count=1 -timeout 6h . 2>&1 | tee /tmp/gotest.log | gotestfmt
```

Comment:
My understanding was without specifying parallelism, Go uses MAX_PARALLELISM=${cpu_count} by default (or 2*cpu_count). Hard to find documentation of this but [this](https://pkg.go.dev/testing#B.RunParallel) is as close as I can get.